frontend/src/components/place/PlaceSidebarInfo.vue

```diff
@@ -82,17 +92,136 @@
 </template>
 
 <script>
+import { mapState, mapGetters, mapActions } from 'vuex';
+import Mapbox from 'mapbox-gl-vue';
+import LocationService from '@/services/location/locationService';
+import markerManager from '@/services/map/markerManager';
+import placeholderImg from '../../assets/placeholder_128x128.png';
+
 export default {
     name: 'PlaceSidebarInfo',
+
+    data() {
+        return {
+            isMapLoaded: false,
+            map: {},
+            userCoordinates: {
+                lat: 50.4547,
+                lng: 30.5238
+            },
+            markerManager: null,
+        };
+    },
+
+    components: {
+        Mapbox
+    },
+
     props: {
         place: {
             type: Object,
             required: true
         }
     },
+
+    created() {
+        if (this.isMapLoaded) {
+            this.updateMap();
+            markerManager.fitMarkersOnMap();
+        }
+    },
+
+    methods: {
+        ...mapActions('place', ['mapInitialization']),
+
+        mapInitialize(map) {
+            if (this.mapInitialized) {
+                return;
+            }
+
+            this.map = map;
+            LocationService.getUserLocationData()
+                .then(coordinates => {
+                    this.userCoordinates.lat = coordinates.lat;
+                    this.userCoordinates.lng = coordinates.lng;
+                });
+            this.mapInitialization();
+        },
+
+        mapLoaded(map) {
+            this.markerManager = markerManager.getService(map);
+            this.isMapLoaded = true;
+        },
+
+        jumpTo(coordinates) {
+            this.map.jumpTo({
+                center: coordinates,
+            });
+        },
+
+        updateMap() {
+            if (this.isMapLoaded) {
+                this.markerManager.setMarkersFromPlacesAndFit(this.place);
+            }
+        },
+        
+        createUserMarker(){
+            return {
+                id           : 0,
+                latitude     : this.userCoordinates.lat,
+                longitude    : this.userCoordinates.lng,
+                localization : {
+                    0: {
+                        description: 'Your position',
+                        language: 'en',
+                        name: 'Your position',
+                    }
+                },
+                photoUrl: this.user.avatar_url || placeholderImg,
+            };
+        }
+    },
+
+    watch: {
+        isMapLoaded: function (oldVal, newVal) {
+            this.updateMap();
+        },
+    },
+
+    computed: {
+        ...mapGetters('map', [
+            'getMapboxToken',
+            'getMapboxStyle'
+        ]),
+        ...mapGetters({
+            user: 'auth/getAuthenticatedUser'
+        }),
+
+        currentCenter() {
+            return {
+                lat: this.place.latitude ? this.place.latitude : this.userCoordinates.lat,
```

Comment:
place always has coordinates, we don't need to  show user's location on that small map